### PR TITLE
update release notes for release 3.3.0

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -60,424 +61,56 @@ with links to every release package along with precompiled binaries.
...
 ----
-wget https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-2.7.0.gem
-gem install couchbase-3.2.0-x86_64-linux-2.7.0.gem
+wget https://packages.couchbase.com/clients/ruby/sdk-3.3.0/couchbase-3.3.0-x86_64-linux-2.7.0.gem
+gem install couchbase-3.3.0-x86_64-linux-2.7.0.gem
 ----
 
 
 [#latest-release]
-== Version 3.2.0 (4 August 2021)
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-release-notes.adoc:69]: @avsej thanks for this! 
Quick one, not sure we should be removing/overriding older releases...
I'll put them back in if that's ok?

Comment (avsej) [reply modules/project-docs/pages/sdk-release-notes.adoc:69]: yes, sure. feel free to put it back.

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -60,424 +61,56 @@ with links to every release package along with precompiled binaries.
...
 | macOS 10.15 x84_64 | 2.7.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.7.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.7.0.gem]
-| macOS 10.15 x84_64 | 2.6.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.6.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.6.0.gem]
-| macOS 10.15 x84_64 | 2.5.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.5.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.5.0.gem]
+| macOS 11 x84_64 | 3.1.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-3.1.0.gem[couchbase-3.2.0-x86_64-darwin-20-3.1.0.gem]
 | macOS 11 x84_64 | 3.0.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-3.0.0.gem[couchbase-3.2.0-x86_64-darwin-20-3.0.0.gem]
 | macOS 11 x84_64 | 2.7.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-2.7.0.gem[couchbase-3.2.0-x86_64-darwin-20-2.7.0.gem]
```
Comment (avsej) [inline modules/project-docs/pages/sdk-release-notes.adoc:115]: the links are not available yet. I will leave comment here once I publish the artifacts to S3

**Review by maria-robobug — APPROVED** [review]

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 2 above)
Comment (maria-robobug) [reply modules/project-docs/pages/sdk-release-notes.adoc:115]: I think It's fine if the docs get published a bit sooner, the links will eventually work once you have published on S3, so I'm merging this 👍 .

**Pull request merged**